packages/realm/src/Results.ts

```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////
 //
-// Copyright 2022 Realm Inc.
+// Copyright 2023 Realm Inc.
```

Comment:
We typically don't update the year in order to establish [prior art](https://en.wikipedia.org/wiki/Prior_art)

Reply:
Very good to know, thanks for pointing it out 🙂